### DISCUSSION TITLE
GitHub showing 3 repositories, I only have 2

### DISCUSSION THREAD

**Discussion by ThePython10110** [community/community#39027] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

For some reason, my GitHub profile says that I have 3 repositories, but only shows 2 (and I only have 2).
If I look at it in a guest window, it says I have 2 repositories but only shows 1 (the numbers are -1 because one of my repositories is private). 

Any idea why this is?
![image](https://user-images.githubusercontent.com/87204246/201549696-5333eebb-7231-451e-ae15-090eee9807b1.png)

**Comment by radar-blip** [comment]
The repo in your Stars category is MuseScores, it also shows up under repositories. 
2 repos + 1 starred = 3 showing

So, its just showing MuseScores twice, yes?

**Comment by ThePython10110** [comment, marked as answer]
For some reason it's back to normal now. I just deleted a repository before starting the discussion, so maybe it takes some time to update that number after deleting a repository? And no, my stars have no effect on it.

**Reply by BeratTezer** [reply]
Directly open this link: https://github.com/ 
You can see all repository which associated with you. (Pull request etc)
